src/file-upload/file-upload.scss

```diff
@@ -39,18 +40,39 @@
       z-index: -1;
     }
 
-    > .iui-icon {
+    > svg {
       @include iui-icons-3xl;
-
-      + span,
-      + label {
-        margin-top: $iui-baseline;
-      }
+      margin-top: $iui-baseline;
+      margin-bottom: $iui-baseline;
     }
 
     &:only-child {
       visibility: visible;
       position: relative;
+      flex-direction: row;
+
+      &:focus-within {
+        outline: 0;
+        @include themed {
+          box-shadow: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-5)) 0 0 0 2px;
+        }
+      }
+
+      > label {
```

Comment:
Maybe we should have a new class for this to not mess with user's labels (that may be unrelated to file input)?

Reply:
Added .iui-label class